Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.17-stretch
+FROM golang:1.18
```

Comment:
Any reason to drop the `-stretch` here? It's going to update us from Debian 9 (Stretch) to Debian 11 (Bullseye) which may be a larger version upgrade than you intended.